AsyncDisplayKit/Private/ASEnvironmentInternal.h

```diff
@@ -35,6 +35,8 @@ void ASEnvironmentPerformBlockOnObjectAndParents(id<ASEnvironment> object, void(
 
 #pragma mark - Merging
 
+static const struct ASEnvironmentStateExtensions ASEnvironmentDefaultStateExtensions = {};
```

Comment:
I assume the answer is yes, but is this format defined to initialize each Field to the correct zero value?